Userland/Libraries/LibJS/Runtime/GlobalObject.cpp

```diff
@@ -350,7 +350,7 @@ static ThrowCompletionOr<String> encode(VM& vm, String const& string, StringView
         auto code_unit = utf16_string.code_unit_at(k);
         // c. If C is in unescapedSet, then
         // NOTE: We assume the unescaped set only contains ascii characters as unescaped_set is a StringView.
-        if (code_unit < 0x80 && unescaped_set.contains(code_unit)) {
+        if (code_unit < 0x80 && unescaped_set.contains((char)code_unit)) {
```

Comment:
Can we avoid c-style casts in new code? static_cast pls (global)

Reply:
Ah, right! I'll change the other casts I introduced.